docs/local.rst

```diff
@@ -0,0 +1,171 @@
+.. _local:
+
+=============
+Local testing
+=============
+
+Further you may find a way to smoketest Singularity-CRI without Kubernetes installation.
+
+----------------------
+Setting up environment
+----------------------
+
+1. Install `crictl <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>`_:
+
+.. code-block:: bash
+
+	VERSION="v1.12.0" && \
+	wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz && \
+	sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin && \
+	rm -f crictl-$VERSION-linux-amd64.tar.gz
+
+2. Configure it work with Singularity-CRI. Create `/etc/crictl.yaml` config file and add the following:
+
+.. code-block:: text
+
+	runtime-endpoint: unix:///var/run/singularity.sock
+	image-endpoint: unix:///var/run/singularity.sock
+	timeout: 10
+	debug: false
+
+For details on all options available see `crictl install page
+<https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md#install-crictl>`_.
+
+3. Build and launch Singularity-CRI server (optionally: :ref:`configure <configuration>`):
+
+.. code-block:: bash
+
+	make clean && \
+	make && \
+	sudo make install && \
+	sudo sycri
+
+
+-----------------------------
+Interact with Singularity-CRI
+-----------------------------
+
+We will walk through a basic examples of interaction with a running Singularity-CRI.
+Further assumed you are located in Singularity-CRI repository root.
+
+Running Nginx
+-------------
+
+1. Run a pod which exposes port 80:
```

Comment:
maybe add that this port is 80 from the default port opening from Nginx, for secure connections is port 443

Reply:
I think for local examples there is not need in this details, port 80 will be used to smoke test CRI